iidm/iidm-criteria/src/main/java/com/powsybl/iidm/criteria/AtLeastOneCountryCriterion.java

```diff
@@ -55,8 +62,10 @@ private List<Country> getCountriesToCheck(Identifiable<?> identifiable, Identifi
         }
     }
 
-    private List<Country> getCountriesToCheck(NetworkElement networkElement) {
-        return Arrays.asList(networkElement.getCountry1().orElse(null), networkElement.getCountry2().orElse(null));
+    private List<Country> getCountriesToCheck(NetworkElement networkElement, ThreeSides side) {
+        return side != null ?
+                Collections.singletonList(networkElement.getCountry(side).orElse(null)) :
+                Arrays.asList(networkElement.getCountry1().orElse(null), networkElement.getCountry2().orElse(null));
```

Comment:
Even if it doesn't really make sense with current equipments, shouldn't we add `getCountry3`? Because your `NetworkElementTest` for instance leads to unexpected behaviour: the `Country.BE` is ignored by this criterion.

Reply:
You are right. I forgot to add it when I introduced `getCountry3()` in `NetworkElement`.